Day1/day1_p1.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const arr = [1721, 979, 366, 299, 675, 1456];
```

Comment:
Where is the code that uses this function, e.g. reads the file, parses the input, passes it to the function?